### PR TITLE
[KOGITO-5723] ForEach Workflow example

### PR DESCRIPTION
**Pull request by fjtirado**
Depends on https://github.com/kiegroup/kogito-runtimes/pull/1754

### CODE REVIEW

File: serverless-workflow-foreach-quarkus/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Kogito Serverless Workflow - Callback Example
```
Comment (ricardozanini) [inline serverless-workflow-foreach-quarkus/README.md]:
```suggestion
# Kogito Serverless Workflow - ForEach Example
```

File: serverless-workflow-foreach-quarkus/README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+### Submit a request
+
+The service based on the JSON workflow definition can be access by sending a request to http://localhost:8080/foreach'
+
+```
+
+Complete curl command can be found below:
+
+```text
+curl -X POST -H 'Content-Type:application/json' -H 'Accept:application/json' -d '{"workflowdata" : {"list": [1,2,3]}' http://localhost:8080/foreach
```
Comment (ricardozanini) [inline serverless-workflow-foreach-quarkus/README.md]:
```suggestion
curl -X POST -H 'Content-Type:application/json' -H 'Accept:application/json' -d '{"workflowdata" : {"input": [1,2,3]}' http://localhost:8080/foreach
```

**Review by ricardozanini — APPROVED** [review]

File: serverless-workflow-foreach-quarkus/pom.xml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    </dependencies>
+  </dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>org.kie.kogito</groupId>
+      <artifactId>kogito-quarkus-serverless-workflow</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.kie.kogito</groupId>  
+      <artifactId>kogito-quarkus-test-utils</artifactId>
```
Comment (tiagodolphine) [inline serverless-workflow-foreach-quarkus/pom.xml]: is this needed? if so, probably the scope should be with test scope.

File: serverless-workflow-functions-events-quarkus/src/main/resources/application.properties [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Packaging
 # quarkus.package.type=fast-jar
-
+quarkus.http.test-port=0
 quarkus.swagger-ui.always-include=true
 mp.openapi.extensions.smallrye.operationIdStrategy=METHOD
 quarkus.log.category."org.acme".level=DEBUG
```
Comment (tiagodolphine) [inline serverless-workflow-functions-events-quarkus/src/main/resources/application.properties:6]: can't it be removed?

File: serverless-workflow-foreach-quarkus/src/test/java/org/kie/kogito/examples/ForEachRestIT.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.http.ContentType;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
+
+@QuarkusTest
+class ForEachRestIT {
```
Comment (tiagodolphine) [inline serverless-workflow-foreach-quarkus/src/test/java/org/kie/kogito/examples/ForEachRestIT.java:29]: maybe adding a native test to this one.

File: serverless-workflow-functions-events-quarkus/src/main/resources/application.properties (hunk 4 above)
Comment (fjtirado) [reply serverless-workflow-functions-events-quarkus/src/main/resources/application.properties:6]: This is needed for that example to work after the quarkus update (actually should be a different JIRA, but just fixing with this one)

Comment (ricardozanini) [reply serverless-workflow-functions-events-quarkus/src/main/resources/application.properties:6]: So that worked :)

**Pull request merged**